packages/ui/src/components/Label/Label.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactText } from 'react';
+import { Box, FormLabel, forwardRef, HTMLChakraProps } from '@chakra-ui/react';
+
+type LabelSizeProps = 'sm' | 'md' | 'lg';
+
+export interface LabelProps extends HTMLChakraProps<'label'> {
+  children: ReactText | ReactText[];
+  size?: LabelSizeProps;
+  required?: boolean;
+}
+
+export const Label = forwardRef<LabelProps, 'label'>(({ size = 'sm', required, children, ...props }, ref) => {
+  return (
+    <FormLabel ref={ref} color="gray.700" fontSize={size} required={required} {...props}>
+      {children}
+      {required ? (
```

Comment:
It can be simplified to `required && <Box as="span" color="red.600">*</Box>`

Reply:
It's not a problem to add it, but one check from vercel is failed - Vercel-coderscamp-docs.

Reply:
You can merge the `main` branch - it will fix the problem with Vercel